lib/ex_doc/autolink.ex

```diff
@@ -184,8 +188,9 @@ defmodule ExDoc.Autolink do
     timeout: 0
   ]
 
-  defp url("mix help " <> name, _mode, config), do: mix_task(name, config)
-  defp url("mix " <> name, _mode, config), do: mix_task(name, config)
+  defp url("", _mode, _config), do: nil
```

Comment:
https://spec.commonmark.org/0.29/#example-482
Title and destination of links can be omitted.
While it works because the last clause will catch it, it will save the trip through that clause

Reply:
I added a raise to that clause and tested with `[foo]()` and the clause wasn’t hit, could you double check?

Reply:
or don’t check but just remove the clause imho :) I don’t think we need to optimize for extremely unlikely cases and thus keep the code simpler

Reply:
removed!

Reply:
ping